zerver/lib/actions.py

```diff
@@ -3642,6 +3642,8 @@ def do_remove_realm_alias(alias):
     realm = alias.realm
     domain = alias.domain
     alias.delete()
+    if RealmAlias.objects.filter(realm=realm).count() == 0:
+        do_set_realm_restricted_to_domain(realm, False)
```

Comment:
Do we need to send an event here to tell the admin system that the realm is no longer restricted?

We should comment this section to make it fairly clear why we flip that boolean.